store/serializers.py

```diff
@@ -611,17 +613,24 @@ def create(self, validated_data):
 
             if membership_orderlines:
                 need_transaction = True
-                today = timezone.now().date()
+                # Allow to buy membership, 1 month before end of membership
+                today = timezone.now().date() + relativedelta(months=1)
```

Comment:
Could be nice to change the name of the variable since it's not today anymore. Could be for example `limit_date`